### PR TITLE
Fixed API names

### PR DESCRIPTION
**Pull request by Yevgnen**

### CODE REVIEW

**Comment by bdarcus** [comment]
Thanks for this @Yevgnen!

Can you please provide a description explaining what you did and why?

I see from your changes there's actually a mistake that you fixed, and maybe some differences of opinion on command names, which we can discuss.

File: bibtex-actions.el [hunk 1]
```diff
@@ -134,17 +134,17 @@ If multiple PDFs are found, ask for the one to open using
...
  (bibtex-completion-insert-bibtex (list keys)))
 
-(defun bibtex-actions-add-PDF-attachment (keys)
+(defun bibtex-actions-add-pdf-attachment (keys)
  "Attach PDF(s) associated with the KEYS to email."
  (interactive (list (bibtex-actions--read)))
  (bibtex-completion-add-PDF-attachment (list keys)))
 
-(defun bibtex-actions-open-notes (keys)
+(defun bibtex-actions-edit-notes (keys)
```
Comment (bdarcus) [inline bibtex-actions.el:142]: See #12 for why I changed this, and the "show" example below.

File: bibtex-actions.el [hunk 2]
```diff
@@ -134,17 +134,17 @@ If multiple PDFs are found, ask for the one to open using
  (interactive (list (bibtex-actions--read)))
  (bibtex-completion-insert-bibtex (list keys)))
 
-(defun bibtex-actions-add-PDF-attachment (keys)
+(defun bibtex-actions-add-pdf-attachment (keys)
```
Comment (bdarcus) [inline bibtex-actions.el:137]: Thanks; not sure how I missed this!

**Comment by Yevgnen** [comment]
Hi, sorry I didn't notice #12 . I tried to fix this only since I found the keymap was broken when dealing with the bibtex completion.

File: bibtex-actions.el [hunk 3]
```diff
@@ -48,9 +48,9 @@
     (define-key map (kbd "r") 'bibtex-actions-insert-reference)
     (define-key map (kbd "k") 'bibtex-actions-insert-key)
     (define-key map (kbd "b") 'bibtex-actions-insert-bibtex)
-    (define-key map (kbd "a") 'bibtex-actions-add-pdf-attachment)
-    (define-key map (kbd "e") 'bibtex-actions-edit-notes)
-    (define-key map (kbd "s") 'bibtex-actions-show-entry)
+    (define-key map (kbd "a") 'bibtex-actions-add-PDF-attachment)
```
Comment (bdarcus) [inline bibtex-actions.el:51]:
```suggestion
    (define-key map (kbd "a") 'bibtex-actions-add-pdf-attachment)
```

**Review by bdarcus — CHANGES_REQUESTED** [review]
You're correction of "PDF" to "pdf" was correct. I fixed that on the keymap; can you update the function name to match (not an easy way for me to do that with the web UI)?

**Comment by Yevgnen** [comment]
@bdarcus Done!

**Review by bdarcus — APPROVED** [review]

**Comment by bdarcus** [comment]
Merged; thanks much!

**Comment by bdarcus** [comment]
PS - if you have time and interest, @Yevgnen, I could use help fixing #18. There's some subtle thing wrong with it ATM.

**Pull request merged**